### DISCUSSION TITLE
blog/schedule-aws-lambda-with-amazon-eventbridge-scheduler/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#3] [General]
# blog/schedule-aws-lambda-with-amazon-eventbridge-scheduler/

Let's learn how to Schedule AWS Lambda with Amazon EventBridge Scheduler using rate and cron expressions. This is part of my ongoing .NET on AWS series, where we explore and learn about various AWS Services that can benefit you, as a .NET Developer. This is yet another piece of article where we learn about the serverless scheduler that Amazon offers. In simple words, Amazon EventBridge Scheduler helps you build event-driven applications, schedule tasks within the AWS ecosystem, and more.

https://codewithmukesh.com/blog/schedule-aws-lambda-with-amazon-eventbridge-scheduler/

**Comment by iammukeshm** [comment]
This is a test comment!
